visualizer/src/Controls/Tracking/Alerts/AlertGroup.js

```diff
@@ -0,0 +1,78 @@
+import { IconButton } from '@material-ui/core';
+import Collapse from '@material-ui/core/Collapse';
+import { makeStyles } from '@material-ui/core/styles';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import Alert from '@material-ui/lab/Alert';
+import clsx from 'clsx';
+import React from 'react';
+
+const useStyles = makeStyles((theme) => {
+  const transition = {
+    duration: theme.transitions.duration.shortest,
+  };
+
+  return {
+    root: {
+      width: '100%',
+      // '& > * + *': {
+      //   marginTop: theme.spacing(2),
+      // },
+    },
+    headerAlert: {
+      boxSizing: 'border-box',
+      maxWidth: '300px',
+      marginTop: theme.spacing(2),
+    },
+    alert: {
+      position: 'relative',
+      boxSizing: 'border-box',
+      maxWidth: '280px',
+      left: '20px',
```

Comment:
Minor: I'm not a big fan of raw pixels, it would be great to use relative units if possible.

Reply:
I switched pixels to rem and I moved these reused styles into AlertGroup.js and imported them in other alerts like OneDaughterAlert.js.

Reply:
Importing the styles really helped clean it up!